README.md

```
@@ -22,33 +22,43 @@ The system follows a 3-tier architecture:
-npm install
-npm start
-```
+To run the web client, backend, and database together in development mode:
 
-Runs on:
+First, make sure you create a `.env` file with your Google API key,
+Plaid Client ID, Plaid (Sandbox) Secret, and AWS credentials.
+
+Example:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)